### PR TITLE
chore: do not check certificate when downloading the elasticsearch plugin

### PR DESCRIPTION
**Pull request by iziang**

### CODE REVIEW

File: addons/elasticsearch/PLEASE_SETUP_KUBECONFIG_MANUALLY [hunk 1]
```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
```
Comment (nayutah) [inline addons/elasticsearch/PLEASE_SETUP_KUBECONFIG_MANUALLY:1]: pls give an empty file

File: addons/elasticsearch/gke_gcloud_auth_plugin_cache [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (nayutah) [inline addons/elasticsearch/gke_gcloud_auth_plugin_cache:1]: pls remove this file

File: addons/elasticsearch/PLEASE_SETUP_KUBECONFIG_MANUALLY (hunk 1 above)
Comment (iziang) [reply addons/elasticsearch/PLEASE_SETUP_KUBECONFIG_MANUALLY]: done

File: addons/elasticsearch/gke_gcloud_auth_plugin_cache (hunk 2 above)
Comment (iziang) [reply addons/elasticsearch/gke_gcloud_auth_plugin_cache]: done

**Review by ldming — APPROVED** [review]

**Pull request merged**